apps/dgst.c

```diff
@@ -413,6 +413,34 @@ int dgst_main(int argc, char **argv)
     return ret;
 }
 
+static const char *newline_escape_filename(const char *file, int *backslash)
+{
+    int i = 0, length = strlen(file), newline_count = 0;
+    while(i < length) {
+        const char c = file[i];
+        if (c == '\n') {
+            newline_count++;
+        }
+        i++;
+    }
+    char *file_cpy = app_malloc(length + newline_count, file);
```

Comment:
Mixed code and declarations isn't allow by the coding standards.

Reply:
Adding this new memory computation on a new line in my new commit.